src/Api/JsonRpc/Client.php

```diff
@@ -447,23 +424,10 @@ private function addTestResultsToTestRun()
 
     private function getCaseRunStatusIdByResult(string $result): int
```

Comment:
Function should be renamed to `getTestExecutionStatusIdByResult` or something like that. `CaseRunStatus` as terminology is long gone and is confusing.